onnxruntime/core/providers/tensorrt/tensorrt_execution_provider.cc

```diff
@@ -3171,9 +3199,7 @@ Status TensorrtExecutionProvider::CreateNodeComputeInfoFromGraph(const GraphView
   }
 
   // Generate file name for dumping ep context model
-  if (dump_ep_context_model_ && ctx_model_path_.empty()) {
-    ctx_model_path_ = GetCtxModelPath(ep_context_file_path_, model_path_);
-  }
+  ctx_model_path_ = GetCtxModelPath(ep_context_file_path_, model_path_);
```

Comment:
I think we should keep the if check, right?
So that we don't need to create the ctx_model_path_ for every EP Context node.

Reply:
You're right, I added it